test/test_doctests.py

```diff
@@ -65,9 +67,12 @@ def load_tests(_loader, tests, _ignore):
                    retorts,
                    azul.plugins.metadata.hca.full_metadata,
                    load_app_module('service'),
+                   load_module(root + '/scripts/check_branch.py', 'check_branch'),
                    load_module(root + '/scripts/envhook.py', 'envhook'),
                    load_module(root + '/scripts/export_environment.py', 'export_environment'),
-                   load_module(root + '/scripts/check_branch.py', 'check_branch'),
+                   load_module(root + '/scripts/prepare_lambda_deployment.py',
+                               'prepare_lambda_deployment'),
```

Comment:
Dislike how this line sticks out. It must be longer than 80 but If you have it in you can you refactor this so the line reads

```suggestion
                   load_script('prepare_lambda_deployment'),
```